installer/install.iss

```diff
@@ -3370,7 +3362,6 @@ begin
         Form.Caption:='Git Uninstall: Removing in-use files';
         Form.ClientWidth:=ScaleX(500);
         Form.ClientHeight:=ScaleY(256);
-        Form.Center;
```

Comment:
Needed to prevent errors in InnoSetup, as discussed in https://github.com/git-for-windows/build-extra/pull/327#discussion_r571471676